src/Graph.ts

```diff
@@ -0,0 +1,174 @@
+/**
+ * Transform array to graph
+ */
+
+/**
+ * TYPES
+ */
+
+type node = {
+  x: Number;
+  y: Number;
+  value: Number;
+  isExit: Boolean;
+};
+
+type edge = {
+  from: string;
+  to: string;
+  direction: string;
+};
+
+type Graph = {
+  nodes: {
+    [coordinates: string]: node;
+  };
+  edges: Array<edge>;
+};
+
+/**
+ * FUNCTIONS
+ */
+
+export function boardToGraph(board: Array<Array<Number>>): Graph | undefined {
```

Comment:
Please define a type for the input board. Something like `type BoardArray = Array<Array<Number>>`

Reply:
Type Board has been added in the [last commit](https://github.com/marmelab/kuba-integration/commit/52c48e332abaf5397cf6debd1512e2111062ba31)